### PR TITLE
modifies stock values for metrics

### PR DESCRIPTION
**Pull request by sudhikan**
Signed-off-by: Sudhi Kandi <sudhikan@fb.com>

[orc8r][metrics]

## Summary

These changes adds the values for the various yml files needed to configure prometheus/alertmanager/elastic search

## Test Plan
Tested on MCP for Carrier WiFi on `28f7f2e8`.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by sudhikan** [comment]
cc: @karthiksubraveti @Scott8440

File: orc8r/cloud/helm/orc8r/charts/secrets/values.yaml [hunk 1]
```diff
@@ -54,10 +54,15 @@ secret:
         profile: "prometheus"
 
         prometheusQueryAddress: "http://orc8r-prometheus:9090"
+        prometheusPushAddresses: "http://orc8r-prometheus-cache:9091/metrics"
```
Comment (Scott8440) [inline orc8r/cloud/helm/orc8r/charts/secrets/values.yaml]: This doesn't need to be in metricsd anymore. It's used in orchestrator.yml

Comment (sudhikan) [reply orc8r/cloud/helm/orc8r/charts/secrets/values.yaml]: ack. Removing it.

File: orc8r/cloud/helm/orc8r/charts/secrets/values.yaml [hunk 2]
```diff
@@ -55,9 +55,13 @@ secret:
...
-        alertmanagerApiURL: "http://orc8r-alertmanager:9093/api/v2/alerts"
-        prometheusConfigServiceURL: "http://orc8r-config-manager:9100"
-        alertmanagerConfigServiceURL: "http://orc8r-config-manager:9101"
+        alertmanagerApiURL: "http://orc8r-alertmanager:9093/api/v2"
+        prometheusConfigServiceURL: "http://orc8r-prometheus-configurer:9100/v1"
+        alertmanagerConfigServiceURL: "http://orc8r-alertmanager-configurer:9101/v1"
+
+      elastic.yml: |-
+        elasticHost: "elasticsearch-master"
+        elasticPort; 9200
```
Comment (karthiksubraveti) [inline orc8r/cloud/helm/orc8r/charts/secrets/values.yaml]: minor nit: ';' -> ':'

**Review by karthiksubraveti — APPROVED** [review]

File: orc8r/cloud/helm/orc8r/charts/secrets/values.yaml (hunk 2 above)
Comment (sudhikan) [reply orc8r/cloud/helm/orc8r/charts/secrets/values.yaml]: Thanks. fixed it.

**Review by xjtian — APPROVED** [review]

**Pull request merged**